code/modules/fishing/fishing_equipment.dm

```diff
@@ -477,5 +477,101 @@
 		Do <b>not</b> spin while the light is still <b>red</b>.<br><br>\
 		That's all, best of luck to your angling journey."
 
+///A modified mining capsule from the black market and sometimes random loot.
+/obj/item/survivalcapsule/fishing
+	name = "fishing spot capsule"
+	desc = "An illegally modified mining capsule containing a small fishing spot connected to some faraway place."
+	icon_state = "capsule_fishing"
+	initial_language_holder = /datum/language_holder/speaking_machine
+	verb_say = "beeps"
+	verb_yell = "blares"
+	voice_filter = "alimiter=0.9,acompressor=threshold=0.3:ratio=40:attack=15:release=350:makeup=1.5,highpass=f=1000,rubberband=pitch=1.5"
```

Comment:
this should probably be an api/component like "verbal_ttsspeaker", but ti's not really something you have to do.

Reply:
This seems like a pretty vague suggestion.

Reply:
what i'm saying is if we're going to let things use TTS to speak, there should be a nice API to enable that with all teh config wrapped up in a nice datum with clear fields and constructor format etc.